configuration.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
I would suggest this file be renamed to `en-GB.json` to specify that the English text is in here. I think `configuration.json` could be confused with any custom user settings.

Reply:
Ok, I agree